### ISSUE TITLE
[BUG]: Tool READMEs do not mention the Toolkit

### ISSUE THREAD

**Issue by tombrunet** [IBMa/equal-access#1818]
## Project

accessibility-checker-engine, accessibility-checker for Node, cypress-accessibility-checker, karma-accessibility-checker

### Description

The READMEs seem to have an assumption that our users discovered the tool via the Equal Access Toolkit, so they don't have any information about how the tool fits into the larger context. At a minimum, the overview paragraph at the top should have a sentence or two mentioning the toolkit and how it (the specific tools for this ReadMe) fits into the larger verification process.

### Links to ReadMe's to be updated:
- [x] [equal-access (root)](https://github.com/IBMa/equal-access/blob/master/README.md) already contains a link to the Toolkit:
> equal-access
> This Git repository hosts tools and supporting components of the [IBM Equal Access Toolkit](https://ibm.com/able/toolkit).
>
> Overview
> This README covers ...
  
#### **_Tools_**
- [x] [accessibility-checker-extension/README](https://github.com/IBMa/equal-access/blob/master/accessibility-checker-extension/README.md): web developer tools browser extensions
- [x] [accessibility-checker/README for Node](https://github.com/IBMa/equal-access/blob/master/accessibility-checker/README.md): Node-based test environments
- [x] [karma-accessibility-checker/README](https://github.com/IBMa/equal-access/blob/master/karma-accessibility-checker/README.md): Karma environment
- [x] [cypress-accessibility-checker/README](https://github.com/IBMa/equal-access/blob/master/cypress-accessibility-checker/README.md): Wrapper for the Cypress environment

#### **_Components_**
- [x] [accessibility-checker-engine/README](https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/README.md): rules and engine used by all the tools
  - [x] [accessibility-checker-engine/README-NPM](https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/README-NPM.md)
  - [x] [accessibility-checker-engine/README-RULES](https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/README-RULES.md)
- [x] [rule-server/README](https://github.com/IBMa/equal-access/blob/master/rule-server/README.md): deploys the rules and engine for usage by the tools

### Steps to reproduce

See description.

**Comment by philljenkins** [comment]
The opening section of each README will include a customized version of the following opening paragraph:

> This Git repository hosts tools and supporting components of the [IBM Equal Access Toolkit](https://ibm.com/able/toolkit). The Toolkit provides the tools and guidance to create experiences that are delightful for people of all abilities. The guidance is organized by phase, such as Plan, Design, Develop, and Verify, and explains how to integrate these automated testing tools into the [Verify phase](https://www.ibm.com/able/toolkit/verify/overview). The Toolkit is a major part of the applications and accessibility information at [ibm.com/able](https://ibm.com/able/).

**Comment by philljenkins** [comment]
Some README's related to Rules include the following:

>### Rules and Rulesets
> 
> * Rules are based on the [IBM Accessibility requirements](https://www.ibm.com/able/requirements/requirements/), which is a unified set of WCAG, EN 301 549, and US 508 standards.
> * Rules are harmonized with the open rules published by the [W3C ACT-Rules Community](https://www.w3.org/community/act-r/) group as reported in the [IBM Equal Access Accessibility Checker ACT implementation report](https://wai-wcag-act-rules.netlify.app/standards-guidelines/act/implementations/equal-access/).
> * Rule sets, such as `IBM Accessibility v7.2`, `WCAG 2.2 (A & AA)`, `WCAG 2.1 (A & AA)`, and `WCAG 2.0 (A & AA)`and mappings of the rules to the standards (Requirements), Rule IDs, the individual failure messages, and links to the Help files are published at [Checker rule sets](https://www.ibm.com/able/requirements/checker-rule-sets).
> * Mappings of the rules are defined in the [individual rule_ID_name.ts files](https://github.com/IBMa/equal-access/tree/master/accessibility-checker-engine/src/v4/rules).
> * Each build creates the `Rules listing` artifact in **Actions** that can be reviewed prior to deployment.

**Comment by philljenkins** [comment]
README's were updated to reference Node Version 18
